test/index.js

```diff
@@ -73,11 +90,10 @@ describe('Deployment', () => {
 
         const options = {
             method: 'POST',
-            url: '/users',
+            url: '/api/users',
```

Comment:
Particular preference of adding this prefix on each route rather than setting it as a registration option (not a plugin option, but hapi's `prefix` registration option)?

Reply:
Oh, I see—this is accounting for a prefix configured for the test!

Reply:
This was actually a hack that I'd forgotten about.  I'd merged @zemccartney 's tandy PR, but never published to npm, and I hadn't logged into npm from my new computer yet, so I was unable to publish at the time I was working through this.  I've now published tandy and removed this hack.